### ISSUE TITLE
Certificate with incorrect line breaks can get fluentbit in a crashloop

### ISSUE THREAD

**Issue by a-thaler** [kyma-project/telemetry-manager#670]
**Description**

A typical scenario is to copy the secret for the backend configuration. Especially for the PEM bodies, you easily can do mistakes with the line breaks. As PEMs are a well-defined format, the manager should try to auto-correct typical mistakes like de-doubling backslashes.

The concrete problem here is a cert file where by mistake the line breaks are escaped (by having double backslashes). Such mistake will bring fluentbit in a crashloop with following error message:

```
[2023/12/22 15:33:53] [error] [tls] crt_file '/fluent-bit/etc/output-tls-config/sap-cloud-logging-application-logs-cert.crt' 336445449: error:0909006C:PEM routines:get_name:no start line
[2023/12/22 15:33:53] [error] [tls] could not create TLS backend
[2023/12/22 15:33:53] [error] [output http.1] error initializing TLS context
[2023/12/22 15:33:53] [error] [engine] output initialization failed
[2023/12/22 15:33:53] [error] [lib] backend failed
```

**Expected result**
It will be great if such mistakes can be self-healed and the setup will just work

**Actual result**
Currently you will need to figure out the problem and fix it manually in the setup

**Criterias**
- A cert/ca/key having escaped line breaks will lead to a functional setup

**Release Notes**

```
Certificates or keys for mTLS communication to the log backend will be sanitized with line break problems
```

**Comment by rakesh-garimella** [comment]
Now we sanitize the certificates passed in the secret and remove extra escape characters.

**Comment by rakesh-garimella** [comment]
Fix for release 1.6.1 https://github.com/kyma-project/telemetry-manager/pull/686
